### PR TITLE
Modules

### PR DESCRIPTION
**Pull request by Eternalgratis**
Awesome books built with modules and ES6

### CODE REVIEW

**Comment by Eternalgratis** [comment]
Screenshot from 2022-07-12 02-41-45

Please kindly view the screenshot above which shows my passed webhint on my terminal but isn't passing on gitHub

**Review by TedLivist — COMMENTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @Eternalgratis,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a project that does not have a descriptive README. Kindly format your documentation using [the Microverse template](https://github.com/microverseinc/readme-template)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+<body>
+    <div class="body">
+        <header>
+            <h2 class="header">Awesome Books</h2>
+            <ul class="nav-link">
+                <li class="nav">List</li>
+                <li class="nav">Add new</li>
+                <li class="nav">Contact</li>
+            </ul>
```
Comment (TedLivist) [inline index.html:27]: - [ ] **[OPTIONAL]** When I go through your nav links, the cursor has the **text pointer** type. Wouldn't it be better to change the cursor to **link pointer** for a better user experience?

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @Eternalgratis,

✔️ Books can be added and removed

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**